randovania/data/json_data/prime1.txt

```diff
@@ -1941,58 +1983,94 @@ Asset id: 1531853045
 > Dock to Temple Security Station; Heals? False
   > Temple Security Station/Dock to Temple Hall
       Trivial
+  > Temple Hall/Dock to Landing Site
+      Trivial
 
 > Dock to Landing Site; Heals? False
   > Landing Site/Dock to Temple Hall
       Trivial
+  > Temple Hall/Dock to Temple Security Station
+      Trivial
 
 ----------------
 Alcove
 Asset id: 3293477383
 > Dock to Landing Site; Heals? False
   > Landing Site/Dock to Alcove
       Trivial
+  > Alcove/Pickup (Space Jump Boots)
+      Trivial
+
+> Pickup (Space Jump Boots); Heals? False
+  > Alcove/Dock to Landing Site
+      Space Jump Boots
 
 ----------------
 Waterfall Cavern
 Asset id: 3882538769
 > Dock to Landing Site; Heals? False
   > Landing Site/Dock to Waterfall Cavern
       Trivial
+  > Waterfall Cavern/Dock to Frigate Crash Site
+      Morph Ball
 
 > Dock to Frigate Crash Site; Heals? False
   > Frigate Crash Site/Dock to Waterfall Cavern
       Trivial
+  > Waterfall Cavern/Dock to Landing Site
+      Morph Ball
 
 ----------------
 Tallon Canyon
 Asset id: 541313390
 > Dock to Canyon Cavern; Heals? False
   > Canyon Cavern/Dock to Tallon Canyon
       Trivial
+  > Tallon Canyon/Half Pipe
+      Trivial
 
 > Dock to Transport Tunnel A; Heals? False
   > Transport Tunnel A/Dock to Tallon Canyon
       Trivial
+  > Tallon Canyon/Half Pipe
+      Trivial
 
 > Dock to Gully; Heals? False
   > Gully/Dock to Tallon Canyon
       Trivial
+  > Tallon Canyon/Half Pipe
+      Trivial
 
 > Dock to Root Tunnel; Heals? False
   > Root Tunnel/Dock to Tallon Canyon
       Trivial
+  > Tallon Canyon/Half Pipe
+      Trivial
+
+> Half Pipe; Heals? False
+  > Tallon Canyon/Dock to Canyon Cavern
+      Trivial
+  > Tallon Canyon/Dock to Transport Tunnel A
+      Trivial
+  > Tallon Canyon/Dock to Gully
+      Can Boost and Can lay bomb
+  > Tallon Canyon/Dock to Root Tunnel
+      Trivial
 
 ----------------
 Temple Security Station
 Asset id: 3182558380
 > Dock to Temple Hall; Heals? False
   > Temple Hall/Dock to Temple Security Station
       Trivial
+  > Temple Security Station/Dock to Temple Lobby
+      Missile
 
 > Dock to Temple Lobby; Heals? False
   > Temple Lobby/Dock to Temple Security Station
       Trivial
+  > Temple Security Station/Dock to Temple Hall
+      Trivial
```

Comment:
This is fine for now, but door locks should be handled by the dock itself and not as a requirement in a node connection.